app/controllers/ems_middleware_controller.rb

```diff
@@ -38,6 +38,13 @@ def ems_middleware_form_fields
     ems_form_fields
   end
 
+  def textual_group_list
+    [
+      %i(properties status),
+      %i(relationships smart_management)
+    ]
+  end
```

Comment:
Why is this here?

Reply:
excellent question :) 
Without this part, it inherits textual_group_list from https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/ems_common.rb#L24
which has topology defined.
And that causes opening several middleware screens to crash.
So overriding it without the topology part solved the problem.